cleaner_engine.py

```
@@ -0,0 +1,1406 @@
+        capabilities = {
+            "cuda": False,
+            "opencl": False,
+            "directx": False
+        }
+        
+        # CUDA-Erkennung
+        if CUPY_AVAILABLE:
+            try:
+                cp.cuda.runtime.getDeviceCount()
+                capabilities["cuda"] = True
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)